docs/contributing/coding-standards/components.md

```diff
@@ -20,8 +20,6 @@ Components must:
 
 Component folder and files should be singular, except in cases where they are more commonly used in groups, for example, radios, breadcrumbs and checkboxes.
 
-An example component exists in `src/components/component-example`.
-
 Use this as the basis for creating new components.
```

Comment:
Should this line be removed as well? 'This' seems to refer to the example component that we've now removed reference to.

Reply:
Yep, I'll update this now.

Great spot 👀